.changeset/ninety-weeks-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Adds missing name/value of the submit button to the form data of a view transition
```

Comment:
```suggestion
Adds properties of the submit button (name, value) to the form data of a view transition
```
I'm tempted not to use "missing" for "feature we just hadn't added yet" (could also be interpreted as "something user neglected to add in their code). We're allowed to say "Adds..." so maybe this instead?